.gitignore

```diff
@@ -72,6 +72,8 @@ instance/
 
 # Sphinx documentation
 docs/_build/
+docs/source/scripts
+docs/source/API_documentation
```

Comment:
What about just `API` it is more intuitive and having another directory `autometa` inside source can also be a bit confusing

Reply:
Maybe @jason-c-kwan will be able to weigh in on this, but I think having `autometa` within the docs is less confusing, since it is a reflection of the `autometa` package in the source tree.

You will know which one you are viewing based on one being located within the `docs` directory and the other being located in the base of the source tree. Similarly, one will contain `.rst` files where the other will contain `.py` files.